packages/client/index.js

```diff
@@ -134,60 +143,70 @@ function capitalize (str) {
 }
 
 // TODO: For some unknown reason c8 is not picking up the coverage for this function
-async function graphql (url, log, headers, query, variables) {
-  const res = await request(url, {
-    method: 'POST',
-    headers: {
-      ...headers,
-      'content-type': 'application/json; charset=utf-8'
-    },
-    body: JSON.stringify({
-      query,
-      variables
+async function graphql (url, log, headers, query, variables, openTelemetry) {
+  const { span, headers: headersWithTelemetry } = openTelemetry ? openTelemetry.startSpanClient(url.toString(), 'POST', headers) : { span: null, headers }
```

Comment:
I think we should bring the `@platformatic/context` `PlatformaticContext` througg

Reply:
I might be missing something here, but how? We don't have a `request` here.